examples/glTFViewer/glTFViewer.cpp

```diff
@@ -273,10 +272,9 @@ namespace {
         constexpr nxt::ShaderStageBit kNoStages{};
         nxt::BindGroupLayout bindGroupLayout = utils::MakeBindGroupLayout(
             device, {
-                        {0, nxt::ShaderStageBit::Vertex, nxt::BindingType::UniformBuffer},
-                        {1, hasTexture ? nxt::ShaderStageBit::Fragment : kNoStages,
+                        {0, hasTexture ? nxt::ShaderStageBit::Fragment : kNoStages,
                          nxt::BindingType::Sampler},
-                        {2, hasTexture ? nxt::ShaderStageBit::Fragment : kNoStages,
+                        {1, hasTexture ? nxt::ShaderStageBit::Fragment : kNoStages,
```

Comment:
It's actually possible to get rid of the entire bind group, but it's kind of annoying without std::optional, so I didn't bother.

Reply:
Ack, np